Makefile

```diff
@@ -207,10 +207,14 @@ CGO_ENABLED := 0
 endif
 export CGO_ENABLED
 
+export GIT_REPO 		 ?= github.com/operator-framework/operator-controller
+export VERSION           ?= $(shell git describe --tags --always --abbrev=0)
+export VERSION_PATH 	 ?= ${GIT_REPO}/internal/version
+export GIT_COMMIT 		 ?= $(shell git rev-parse HEAD)
```

Comment:
Don't use `?=`, especially with shell commands. Because they run every time a variable is referenced. See the changes I made here: https://github.com/operator-framework/operator-controller/pull/813

Also, I see a mix of spaces and tabs. @m1kola had a good point when he [reverted](https://github.com/operator-framework/operator-controller/pull/827) the vertical spacing I added in the above PR, so let's avoid that here as well.